src/main/java/com/rusticisoftware/cloud/v2/client/api/AboutApi.java

```diff
@@ -35,8 +35,8 @@ public void setApiClient(ApiClient apiClient) {
   }
 
   /**
-   * Get back the version and platform of the running instance of Engine
-   * 
+   * Get back the API version and application name. 
+   * Get back the API version and application name. 
```

Comment:
Looks like this was duplicated (would this be a product of the code generation or not?)